views/pages/login.ejs

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
```

Comment:
@Mibrer

Reply:
Yes you could delete them.
I was trying to make ejs, but since just 1st iter, turn to html instead

All I make is necessary is  login.html  register.html , 2 js file and index.js

Reply:
Ok if you intend to use them later thats fine then